src/web/View.tsx

```diff
@@ -66,10 +66,12 @@ const _styles = {
 
 if (typeof document !== 'undefined') {
     const ignorePointerEvents = '.reactxp-ignore-pointer-events  * { pointer-events: auto; }';
+    const blockPointerEvents = '.reactxp-block-pointer-events * { pointer-events: none !important; }';
```

Comment:
Is the usage of `!important` needed here?

Reply:
Adding the ".reactxp-block-pointer-events" CSS rule to the document after the ignorePointerEvents rule may be sufficient to have a parent with it applied take precedence over any descendant with the ignorePointerEvents rule set (since both CSS rules otherwise have the same specificity).

However, its a bit fragile. The usefulness of blockPointerEvents is that you can easily apply it to a parent and make the entire subtree transparent to the mouse, regardless of any pointer-events settings on those children, and regardless of whether those children are ReactXP built-ins or external (e.g. SVG). Without !important, its a bit too easy for an inline-style or slightly more specific CSS rule to override that behavior and start making elements opaque again. At least with the !important rule, you need another !important rule to override it.